src/Fetch.js

```diff
@@ -554,7 +570,18 @@ function emscripten_start_fetch(fetch, successcb, errorcb, progresscb) {
   }
   return fetch;
 #else // !FETCH_SUPPORT_INDEXEDDB
-  __emscripten_fetch_xhr(fetch, reportSuccess, reportError, reportProgress);
+  __emscripten_fetch_xhr(fetch, reportSuccess, reportError, reportProgress, reportReadyStateChange);
   return fetch;
 #endif // ~FETCH_SUPPORT_INDEXEDDB
 }
+
+function _emscripten_fetch_get_response_headers_length(id) {
+    return lengthBytesUTF8(Fetch.xhrs[id-1].getAllResponseHeaders()) + 1;
+}
+
+function _emscripten_fetch_get_response_headers(id, dst, dstSizeBytes) {
```

Comment:
Let's rename these two from `_emscripten_fetch_get_response_headers_length` and `_emscripten_fetch_get_response_headers` to `_fetch_get_response_headers_length` and `_fetch_get_response_headers`. The rationale is that the prefix `emscripten_*` is used for public facing APIs, and underscore `_` is used for private APIs - so mixing these two meanings with `_emscripten_*` creates a bit of a cognitive clash with these two meanings. Generally for private APIs we have used shorter underscore-prefixed names local to the library in question. (and when compiled, they will then turn into double underscores to conform to the C spec, i.e. `__fetch_get_response_headers()`)

Reply:
Done.

Reply:
Realised I changed the name in some wrong places - should be fixed now.